cadc-vos-server/src/main/java/ca/nrc/cadc/vos/server/web/restlet/action/NodeAction.java

```diff
@@ -114,6 +114,8 @@
     public static final String QUERY_PARAM_DETAIL = "detail";
     protected static final String QUERY_PARAM_URI = "uri";
     protected static final String QUERY_PARAM_LIMIT = "limit";
+    protected static final String QUERY_SORT_KEY = "sort";
```

Comment:
Names of these constants don't follow the style from previous lines...

Reply:
Done